lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -145,62 +169,33 @@ module.exports = [
                 node: {
                   type: 'node',
                   nodeLabel: 'img',
-                  path: '2,HTML,1,BODY,0,IMG',
+                  selector: 'body > div#late-content > img',
                 },
               },
             ],
           },
         },
-      },
-    },
-  },
-  {
-    lhr: {
-      requestedUrl: 'http://localhost:10200/perf/fonts.html',
-      finalUrl: 'http://localhost:10200/perf/fonts.html',
-      audits: {
-        'font-display': {
-          score: 0,
+        'layout-shift-elements': {
+          score: null,
+          displayValue: '2 elements found',
           details: {
             items: {
               length: 2,
             },
           },
         },
+        'long-tasks': {
+          score: null,
+          details: {
+            items: {
+              0: {
+                url: 'http://localhost:10200/perf/delayed-element.js',
+                duration: '>500',
+              },
+            },
+          },
+        },
       },
     },
   },
-  // TODO: uncomment when Chrome m83 lands
```

Comment:
I'm ok with removing this but can you make an issue + assign to yourself so we don't forget to add it back when m84 (comment is wrong) lands?